paddle/phi/kernels/onednn/concat_kernel.cc

```diff
@@ -65,13 +65,13 @@ class ConcatOneDNNHandler : public OneDNNHandlerNoCachingT<T, dnnl::concat> {
 
   // (jczaja) concat oneDNN prim is not having .desc attribute so
```

Comment:
Please remove comment and commented code

Reply:
fixed